lib/msal-core/.eslintrc.json

```diff
@@ -26,8 +26,8 @@
         "@typescript-eslint/no-object-literal-type-assertion": 0,
         "@typescript-eslint/prefer-interface": 0,
         "@typescript-eslint/type-annotation-spacing": 0,
-        "no-var": 0,
-        "prefer-const": 0,
+        "no-var": 2,
+        "prefer-const": 2,
```

Comment:
Looks like this is doing both rules, I'm guessing you created this branch from the other branch?

Reply:
I think pulling master into this branch will clean this up.

Reply:
oh snap, updated